src/peergos/server/corenode/IpfsCoreNode.java

```diff
@@ -30,6 +31,7 @@
     private final MutablePointers mutable;
     private final SigningPrivateKeyAndPublicHash signer;
 
+    private final AtomicInteger difficulty = new AtomicInteger(11);
```

Comment:
It's just the initial value, so doesn't really matter. I'm probably going to change it to 0 anyway, because it gets adjusted very quickly based on query rate.